spec/tictactoesetup_spec.rb

```diff
@@ -0,0 +1,102 @@
+require 'spec_helper'
+require 'tictactoe_setup'
+require 'fake_io'
+
+class FakeUserInput
+  def initialize(io:)
+    @io = io
+  end 
+end
+class FakePresenter; end
+class FakeTicTacToeSetup
+  attr_reader :create_two_human_players_called,
+              :create_human_and_computer_player_called
+
+  def initialize(user_input:)
+    @create_two_human_players_called = false
+    @create_human_and_computer_player_called = false
+  end
+
+  def build(mode:, presenter:)
+    @create_two_human_players_called = true if mode == 'human vs. human'
+    @create_human_and_computer_player_called = true if mode == 'human vs. computer'
+  end
+
+  def create_two_human_players; end
+
+  def create_human_and_computer_player; end
+end
+
+class FakeMenu
+  attr_accessor :options
+  def initialize(user_input:)
+    @options = []
+  end
+
+  def option_at(position:)
+    @options[position - 1]
+  end
+
+  def add_option(option:)
+    options << option
+  end
+end
+
+describe TicTacToeSetup do
+  context 'when the game menu is created' do
+    it 'has a human vs human option' do
+      setup = TicTacToeSetup.new(user_input: FakeUserInput.new(io: FakeIO.new))
+      menu = setup.create_game_mode_menu
+      game_menu_option = menu.option_at(position: 1)
+      expect(game_menu_option).to eql 'human vs. human'
+    end
+
+    it 'has a human vs computer option' do
+      setup = TicTacToeSetup.new(user_input: FakeUserInput.new(io: FakeIO.new))
+      menu = setup.create_game_mode_menu
+      game_menu_option = menu.option_at(position: 2)
+      expect(game_menu_option).to eql 'human vs. computer'
+    end
+  end
+
+  context 'when the game data is built' do
+    it 'returns game data'  do
+      setup = TicTacToeSetup.new(user_input: FakeUserInput.new(io: FakeIO.new))
+      game_data = setup.build(mode: 'human vs. human',
+                              presenter: FakePresenter.new)
+      expect(game_data).not_to be_kind_of NilClass
```

Comment:
I think this could be simplified to 

```ruby
expect(game_data).not_to be nil
```

I would advise that you be cautious of you the `not_to`. In my experience, it can easily offer a false sense of security.